### DISCUSSION TITLE
umi4 global.less文件不执行

### DISCUSSION THREAD

**Discussion by lbyupup** [umijs/umi#9254] [Q&A]
升级 umi4 后，global.less 文件在本地开发不会加载，只有修改到该文件保存后才会加载样式，刷新页面后样式又不再加载。
打包后都不会执行 global.less 文件。

**Comment by lbyupup** [comment]
https://github.com/umijs/umi/issues/8326
看上去是样式层叠了，持续关注一下

**Reply by fz6m** [reply]
通过关闭 `mfsu: false` 尝试解决该问题。

**Reply by lbyupup** [reply]
> 通过关闭 `mfsu: false` 尝试解决该问题。

我给样式加上 `!important` 之后样式就出来了，应该是样式被层叠了
